src/boot/bootctl.c

```diff
@@ -1647,6 +1653,18 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_arch_all = true;
                         break;
 
+                case ARG_EFI_LABEL:
+                        r = free_and_strdup(&arg_efi_label, optarg);
+                        if (r < 0)
+                                return log_oom();
+                        if (isempty(arg_efi_label) || !string_is_safe(arg_efi_label))
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                       "Invalid --efi-label=: %s", arg_efi_label);
```

Comment:
If the string is not safe, the string should not be directly passed to the logger.
Please `cescape()` the string and log it.
E.g. see `config_parse_string()` in shared/conf-parser.c.